### ISSUE TITLE
ZeroDivisionError

### ISSUE THREAD

**Issue by wavymulder** [neonbjb/DL-Art-School#2]
I'm attempting to get DLAS training ESRGAN. I'm on Windows 10 and using Anaconda.

Here's what I'm getting:
```
Traceback (most recent call last):
  File "train.py", line 361, in <module>
    trainer.init(args.opt, opt, args.launcher)
  File "train.py", line 114, in init
    self.total_epochs = int(math.ceil(total_iters / train_size))
ZeroDivisionError: division by zero
```

Not sure how to fix my train_size being zero. Is my path to my images wrong? Thanks.

**Comment by wavymulder** [comment]
Yup my path was wrong, my mistake sorry!

**Comment by neonbjb** [comment]
Hey there,
I'm really sorry, but I cannot spare the effort to actually support DLAS for wider use. Despite the fact that I started down the road of writing documentation and providing "recipes" like exists for ESRGAN, I don't maintain a test suite that ensures that these things don't get broken and it would be impossible for me to ensure backwards compatibility.

With that being said, I believe what is happening in your case is you've misconfigured the dataset somehow such that train_size=0. I'd set num_workers=0 and try adding some breakpoints into single_image_dataset.py to try and figure out what is going on.

It has also been some time since I've trained a GAN on DLAS. If you're interested in actually getting this to work, I would highly recommend reverting to the commit where I added the ESRGAN recipe: bbc677dc7bb5d3ad32e3754198eecb156093051a
